numalogic/transforms/_movavg.py

```diff
@@ -27,21 +27,6 @@ def _allow_only_single_feature(data: npt.NDArray[float]) -> None:
         )
 
 
-def tanh_norm(scores: npt.NDArray[float], scale_factor=10, smooth_factor=10) -> npt.NDArray[float]:
-    """
-    Applies column wise tanh normalization to the input data.
-
-    This is most commonly used to normalize the anomaly scores to a desired range.
-
-    Args:
-    ----
-        scores: feature array
-        scale_factor: scale the output by this factor (default: 10)
-        smooth_factor: factor to broaden out the linear range of the graph (default: 10)
-    """
-    return scale_factor * np.tanh(scores / smooth_factor)
-
-
 def expmov_avg_aggregator(
```

Comment:
cant we move expmov_avg_aggregator and tanh_norm to the respective classes?

Reply:
So both of those are functional API equivalents of their respective classes. This is similar to [functional conv1d](https://pytorch.org/docs/stable/generated/torch.nn.functional.conv1d.html) and [](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html#torch.nn.Conv1d).
The reason we have functional API, is that these transforms are stateless, and can be used as such.

Once we have more transforms, we will add into a stand-alone `functional` module.